velib_spot_predictor/utils.py

```diff
@@ -0,0 +1,34 @@
+"""Utility functions for the velib_spot_predictor package."""
+from pathlib import Path
+
+
+def get_one_filepath(folder: Path, pattern: str) -> Path:
```

Comment:
Fonction utilisée uniquement par FolderExtractor dans une méthode non utilisée (cf review de publish.py) à voir si c'est utile à garder ?

Reply:
I'll keep it for now, I still find it useful, but thanks for the investigation.